Labs/module-1_labs/03_mysql/lab_sql-extras/olist_data/exercises/lab-1-olist.md

```diff
@@ -0,0 +1,153 @@
+![IronHack Logo](https://s3-eu-west-1.amazonaws.com/ih-materials/uploads/upload_d5c5793015fec3be28a63c4fa3dd4d55.png)
+
+# Lab | SQL
+
+## Introduction
+
+Run your first SQL commands! In this lab, we will practice selecting and projecting data. You can finish all questions with only those clauses:
+- `SELECT`
+- `SELECT DISTINCT`
+- `COUNT`
+- `FROM`
+- `WHERE`
+- `ORDER BY`
+- `GROUP BY`
+- `SUM`
+- `LIMIT`
+
+Please submit your solutions in a text file `solutions.sql`.
+
+### 1. From the `order_items` table, find the price of the highest priced order and lowest price order.
+
+Result:
+```
+6735
+0.85
+```
+Solution:
+```sql
+SELECT *
+FROM olist.order_items
+ORDER BY price DESC
+LIMIT 100;
+
+SELECT *
+FROM olist.order_items
+ORDER BY price
+LIMIT 100;
+```
```

Comment:
Ideally, every problem should be solvable using one query. Either split this exercise into two, or this could require a `UNION` clause to solve. However, we haven't discussed `UNION`s (but maybe we should)

Reply:
Actually, I'd say the most straightforward solution is:
```sql
SELECT
    MIN(price)  AS min_price,
    MAX(price) AS max_price
FROM olist.order_items;
```